Modulo1-Fundamentos/6-HTML_e_CSS:_Forms,_Flexbox_e_Responsivo/6.1-HTML-&-CSS-Forms/exercicios/style.css

```
@@ -0,0 +1,101 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+#heder {
+    left: 0px;
+    top: 0px;
+    right: 0px;
+    width: 100%;
+    margin-right: 0px;
+    margin-left: 0px;
+    padding: 10px 0px;
+    border-bottom: 7px solid #2fc18c;
+    background-color: #036b52;
+    color: #fff;
+}
+
+#titulo {
+    margin-left: 28%;
+    display: inline-block;
+}
+
+#titulo h1 {
+    padding: 0px;
+    margin: 0px;
+    margin-bottom: 10px;
+}
+
+.image-45 {
+    display: inline-block;
+    margin-left: 40px;
+    width: 84px;
+    height: 50px;
+    max-width: 80%;
+}
+
+#harerMain{
+    margin-top: 0px;
```

```suggestion
    margin-top: 0;
```
É um pequeno detalhe, mas sempre que o valor for 0 você pode ocultar a unidade pixels ;D